src/schema/v2/artwork/artworkConsignmentSubmissionType.ts

```diff
@@ -53,6 +54,41 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
           return ["submitted", "hold", "open"].includes(state?.toLowerCase())
         },
       },
+      state: {
+        type: GraphQLString,
+        description: "Submission state.",
+        resolve: ({ state }) => state.toLowerCase(),
```

Comment:
Why the `toLowerCase`? I think this doesn't change anything at the moment, and if it did introduce a difference with the back-end in the future, it would get confusing.

Q: Should this be an enum, for better self-documentation?

Reply:
Removed `toLowerCase`. There was some logic behind it but I can't remember why I did this now.

Reply:
Now I'm confused about why the values are all uppercase... I assumed they'd come from Convection as lowercase, like in the raw data. I think I'm missing some in-between step.

Reply:
Note that this happened too on amber with `visibility_level`. There's some strange uppercase / lowercase thing going on that I don't quite understand.